docs_src/microservices/device/supported/device-usb-camera/walkthrough/deployment.md

```diff
@@ -109,6 +109,94 @@ Follow this guide to deploy and run the service.
     !!! Note 
         The `jq -r` option is used to reduce the size of the displayed response. The entire device with all information can be seen by removing `-r '"deviceName: " + '.devices[].name'', and replacing it with '.'`
 
+1. Add credentials for the rtsp stream.
```

Comment:
The number listing has to be fixed in this page, at least that's what I see in my IDE.

Reply:
When I render it, the numbers look fine

Reply:
Yep, as long as they are just numbers, even if they are out of order, it will render them in order.